src/main/features/claim/helpers/eligibility/validationGroups.ts

```diff
@@ -9,4 +9,6 @@ export class ValidationGroups {
   static readonly DEFENDANT_ADDRESS: string = 'defendant-address'
   static readonly GOVERNMENT_DEPARTMENT: string = 'government-department'
   static readonly CLAIM_IS_FOR_TENANCY_DEPOSIT: string = 'claim-is-for-tenancy-deposit'
+  static readonly DEFENDANT: string = 'defendant'
```

Comment:
I would either revert the change or put these constants in different place. Eligibility validation groups does not look as the right place for it.